mobile/cmp/navigator/NavigatorModel.ts

```diff
@@ -44,6 +44,11 @@ export interface NavigatorConfig {
      * See enum for description of supported modes.
      */
     refreshMode?: RefreshMode;
+
+    /**
+     * Base route name for this navigator, with the route for each page being "[route]/[page.id]".
+     */
+    route?: string;
```

Comment:
Matching the config from `tabContainerModel` - allow dev to scope down routing responsiveness to a particular sub-path.